lib/graphql/schema/member/accepts_definition.rb

```diff
@@ -37,55 +37,85 @@ class Member
       #   Account.graphql_definition.metadata[:permission_level] # => 1
       module AcceptsDefinition
         def self.included(child)
-          child.extend(ClassMethods)
+          child.extend(AcceptsDefinitionClassMethods)
           child.prepend(ToGraphQLExtension)
           child.prepend(InitializeExtension)
         end
 
         def self.extended(child)
-          child.extend(ClassMethods)
-          # I tried to use `super`, but super isn't quite right
-          # since the method is defined in the same class itself,
-          # not the superclass
-          child.class_eval do
-            class << self
-              prepend(ToGraphQLExtension)
+          if defined?(child::ClassMethods)
+            child::ClassMethods.include(AcceptsDefinitionClassMethods)
+            child::ClassMethods.prepend(ToGraphQLExtension)
+          else
+            child.extend(AcceptsDefinitionClassMethods)
+            # I tried to use `super`, but super isn't quite right
+            # since the method is defined in the same class itself,
+            # not the superclass
+            child.class_eval do
+              class << self
+                prepend(ToGraphQLExtension)
+              end
             end
           end
         end
 
-        module ClassMethods
+        module AcceptsDefinitionClassMethods
```

Comment:
Here was a weird naming conflict with `ClassMethods`, case in point for renaming `Schema::Interface`'s inherited module to `DefinitionMethods`